src/components/select/select.template.tsx

```
@@ -50,27 +50,41 @@
+    return [
+        <limel-notched-outline
+            class={classList}
+            labelId="s-label"
+            label={props.label}
+            required={props.required}
+            invalid={!isValid}
+            disabled={props.disabled}
+            readonly={props.readonly}
+            hasValue={hasValue}
+            hasFloatingLabel={floatLabelAbove(props)}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)